rest/messages/generate-twiml-mms/generate-twiml-mms.6.x.php

```diff
@@ -5,7 +5,7 @@
 use Twilio\TwiML\MessagingResponse;
 
 $response = new MessagingResponse;
-$message = $response->message();
+$message = $response->message("");
```

Comment:
why does this change?

Reply:
Without the quotes as an argument, I get this error:
```text
❯ php respond-text.php
PHP Fatal error:  Uncaught ArgumentCountError: Too few arguments to function Twilio\TwiML\MessagingResponse::message(), 0 passed in /Users/shawnstern/coding/track-jacket-app/respond-text.php on line 8 and at least 1 expected in /Users/shawnstern/coding/track-jacket-app/vendor/twilio/sdk/src/Twilio/TwiML/MessagingResponse.php:27
Stack trace:
#0 /Users/shawnstern/coding/track-jacket-app/respond-text.php(8): Twilio\TwiML\MessagingResponse->message()
#1 {main}
  thrown in /Users/shawnstern/coding/track-jacket-app/vendor/twilio/sdk/src/Twilio/TwiML/MessagingResponse.php on line 27

Fatal error: Uncaught ArgumentCountError: Too few arguments to function Twilio\TwiML\MessagingResponse::message(), 0 passed in /Users/shawnstern/coding/track-jacket-app/respond-text.php on line 8 and at least 1 expected in /Users/shawnstern/coding/track-jacket-app/vendor/twilio/sdk/src/Twilio/TwiML/MessagingResponse.php:27
Stack trace:
#0 /Users/shawnstern/coding/track-jacket-app/respond-text.php(8): Twilio\TwiML\MessagingResponse->message()
#1 {main}
  thrown in /Users/shawnstern/coding/track-jacket-app/vendor/twilio/sdk/src/Twilio/TwiML/MessagingResponse.php on line 27
```

I'm guessing that the v6 version requires an input, even if it's just an empty string. I assumed this would work, and also saw this approach in another doc that uses v6, so I assume it is at least viable if not the "right" approach.

https://www.twilio.com/docs/messaging/twiml?code-sample=code-send-hello-world-and-redirect-twiml-control&code-language=PHP&code-sdk-version=6.x